packages/build/src/steps/run_core_step.js

```diff
@@ -0,0 +1,13 @@
+import { coreStep } from '../plugins_core/edge_functions/index.js'
+
+const runCoreStep = async (bundleType, options) => {
```

Comment:
Okay, it makes sense to mirror `buildSite`'s signature in `runCoreStep`. If the only step we want to run is the bundling step, would that mean calling and passing options to [`runSteps`](https://github.com/netlify/build/blob/main/packages/build/src/steps/run_steps.js#L14) from `runCoreStep` instead of directly calling `coreStep`?